src/is_face_detector/main.py

```diff
@@ -0,0 +1,35 @@
+from utils import load_options, get_np_image, get_pb_image
+from is_wire.core import Logger, Channel, Subscription, Message
+from faceDetection import FaceDetector
+from is_msgs.image_pb2 import Image
+import cv2
+import time
+import re
+from options_pb2 import FaceDetectorOptions 
+
+def main():
+
+    service_name = "FaceDetector.Detection"
+    log = Logger(name=service_name)
+    op = load_options()
+    k = FaceDetector()
+    re_topic = re.compile(r'CameraGateway.(\w+).Frame')
+    channel = Channel(op.broker_uri)
+    subscription = Subscription(channel=channel, name=service_name)
+    subscription.subscribe(topic='CameraGateway.*.Frame')
+
+
+    while True:
+        msg= channel.consume()
+        im = msg.unpack(Image)
+        im_np = get_np_image(im)
```

Comment:
where is the import of this function?
Maybe it's better to create a file named image-tools.py, rename the functions `get_np_image` to `to_np` and use it like that:
```python
import image_tools as import img
img.to_np(im)
```

It must not be exactly as I said, but think about some changes like so.